compose.yml

```diff
@@ -1,7 +1,10 @@
 version: '1'
 services:
-  # neo4j-store:
-  #   build: "./data/"
+  neo4j-dbms:
```

Comment:
We should set the default username/password for this instance, maybe drag the values from an env file?

We should also create a volume for neo4j data so that we can persist our graphs between invocations of docker compose up/down

Reply:
username/password defined in the .env file under /data now 👍